tools/benchmark/src/mocha/customOutputRunner.ts

```diff
@@ -65,6 +65,12 @@ export function benchmarkCustom(options: CustomBenchmarkOptions): Test {
 /**
  * Allows the benchmark code to report custom measurements.
  *
+ * @see {@link benchmarkCustom}
+ * @see {@link CustomBenchmarkOptions.run}
+ *
+ * @see {@link benchmarkCustom}
+ * @see {@link CustomBenchmarkOptions.run}
+ *
```

Comment:
Not opposed to adding this to docs but a) dedupe them, and b) this will only be reflected in the next release of benchmark-tool, and on its own doesn't merit a new release IMO. They can stay there for whenever something actually warrants a new release.